conf/pfdns.conf.example

```diff
@@ -21,6 +21,13 @@ proxy . /etc/resolv.conf
     }
     # Anything not handled by pfdns will be resolved normally
 [% domain %]
+
+    # This configuration will check in the /etc/hosts and if the fqdn match it will return the corresponding ip.
+    # If it doesn't match then it will continue
+    #hosts {
```

Comment:
Shouldn't that be enabled by default so that this works out of the box ?

Otherwise we would need to point to this somewhere in the doc for inline

Reply:
Yes, we can enable it by default. I can't see any issue with that.
I mean that if you are at this step in pfdns it means that your device is registered and you want to add custom dns entries (like internal).

Reply:
In that case lets go with that so that it works out of the box